tests/Aspire.Hosting.Tests/Utils/TestDistributedApplicationBuilder.cs

```diff
@@ -61,7 +61,7 @@ public static TestDistributedApplicationBuilder Create(Action<DistributedApplica
         return new TestDistributedApplicationBuilder(configureOptions, testOutputHelper);
     }
 
-    public static TestDistributedApplicationBuilder CreateWithTestContainerRegistry(ITestOutputHelper testOutputHelper) =>
+    public static TestDistributedApplicationBuilder CreateWithTestContainerRegistry(ITestOutputHelper? testOutputHelper = null) =>
```

Comment:
I think this should be reverted. We always want an ITestOutputHelper to be passed so we get log output correctly.

Reply:
I was debugging 😄